CHANGELOG.md

```diff
@@ -16,6 +21,9 @@ tolerance value to be user-specified.
 - Changed the behavior of checking of telescope location to look at the combination of
 `antenna_positions` and `telescope_location` together for `UVData`, `UVCal`, and `UVFlag`.
 Additionally, failing this check results in a warning (was an error).
+- Changed `UVData.write_ms` to sort polarizations based on CASA-preferred ordering.
+- Added some functionality to the `utils._convert_to_slices` method to enable quick
+assessment of whether an indexing array can be replaced by a single slice.
```

Comment:
This should be at the top (reverse chronological).

Reply:
Fixed in 9c862da2